app/client/jsx/Welcome.jsx

```diff
@@ -113,15 +122,42 @@ class Welcome extends Component {
         if (this.state.username) { avatar_opacity = 'form' }
         if (this.state.username && this.state.avatar) { party_opacity = 'form-party' }
 
+        const config = Config.welcomePage
+        const bgStyle = {
+            backgroundColor: config.backgroundColor
+        }
+
+        const splash = config.backgroundImagePath
+            ? <img className="splash" src={config.backgroundImagePath}/>
+            : <div className="splash" style={bgStyle}></div>
+
+        const avatarSelect = this.avatarSelectionEnabled
+            ? <PuckSelect opacity={avatar_opacity} handleSelect={this.handleAvatarSelect.bind(this)} />
```

Comment:
FYI @danielmdavis or @SeanMcGrath if you want to change it in this PR, `avatar_opacity`, `name_opacity`, `text_entry` etc should be camel case